src/storage/mapping/SubdomainExtensionBasedMapper.ts

```diff
@@ -36,7 +36,7 @@ export class SubdomainExtensionBasedMapper extends ExtensionBasedMapper {
   private readonly baseParts: { scheme: string; rest: string };
 
   public constructor(base: string, rootFilepath: string, baseSubdomain = 'www',
-    overrideTypes = { acl: TEXT_TURTLE, meta: TEXT_TURTLE }) {
+    overrideTypes: Record<string, string> = { acl: TEXT_TURTLE, meta: TEXT_TURTLE, trig: APPLICATION_TRIG }) {
```

Comment:
I definitely do not like this duplication here. Let's export this object from `./../util/ContentTypes`?
Then I also have no objection for using it as the default in `ExtensionBasedMapper` above.

Reply:
We could also just remove the default values from the constructor actually since they're in the config anyway.

Reply:
Yeah, but the problem is that these values need to be encoded multiple times in config.